larray/io/excel.py

```diff
@@ -105,6 +79,9 @@ def __init__(self, filepath=None, overwrite_file=False, visible=None, silent=Non
                     if not ext.startswith('.xl'):
                         raise ValueError("'%s' is not a supported file "
                                          "extension" % ext)
+                    if not os.path.isfile(filepath) and not overwrite_file:
+                        raise ValueError("File {} does not exist. Please give the path to an existing file "
+                                         "or set mode to 'w' or 'a'".format(filepath))
```

Comment:
this needs to be changed too (yes I know, this is all my fault, please do not hate me too much ;-))